pycloudlib.toml.template

```diff
@@ -71,6 +71,8 @@ domain_name = ""
 config_path = "~/.oci/config"
 availability_domain = ""  # Likely in ~/.oci/oci_cli_rc
 compartment_id = ""  # Likely in ~/.oci/oci_cli_rc
+# region = "us-phoenix-1"  # will use region from oci config file if not specified
```

Comment:
Do we think it is worth adding docs for this new configs to https://pycloudlib.readthedocs.io/en/latest/clouds/oci.html ?

Reply:
Normally, I would say yes the docs should be updated as well, but in this case, the oracle docs seem woefully out of date and far too barebones so I think this should be done in a separate PR. I will put up an issue for this and fix it myself sometime soon. S

Reply:
Thanks! Could you please link the issue here when done?